src/content/configuration/configuration-languages.md

```diff
@@ -118,7 +118,6 @@ config =
     use: 'babel-loader'
   } ]
   plugins: [
-    new (webpack.optimize.UglifyJsPlugin)
```

Comment:
shouldn't we add `Terser` here instead?

Reply:
This is explained in the commit message :-)

Reply:
Aw sorry didn't see it. Yeah makes sense, thanks!